edx_arch_experiments/datadog_monitoring/code_owner/utils.py

```diff
@@ -88,7 +88,9 @@ def get_code_owner_mappings():
     # .. setting_description: Used for monitoring and reporting of ownership. Use a
     #      dict with keys of code owner name and value as a list of dotted path
     #      module names owned by the code owner.
-    code_owner_mappings = getattr(settings, 'CODE_OWNER_MAPPINGS', {})
+    code_owner_mappings = getattr(settings, 'CODE_OWNER_MAPPINGS', None)
+    if code_owner_mappings is None:
+        return None
```

Comment:
**Note for Reviewer:** A bug was found in this code, because there was a bug in the original test for this that didn't catch this problem.

Reply:
What impact would this have had?

Reply:
When I back out this change, I now get the following unit test failure:
```
FAILED edx_arch_experiments/datadog_monitoring/tests/code_owner/test_middleware.py::CodeOwnerMetricMiddlewareTests::test_code_owner_no_mappings - AssertionError: Expected 'set_custom_attribute' to not have been called. Called 1 times.
```
With the bug, rather than exiting early, we get a custom attribute for `code_owner_2_module`.